### ISSUE TITLE
Torpedoes

### ISSUE THREAD

**Issue by Unicide** [Bubbus/ACF-Missiles#52]
Torpedoes! They would essentially function as slow, fuel-efficient large unguided rockets that remain on the surface of the water (or perhaps at some set depth below). Naval combat is a pretty niche thing in ACF, but adding torpedoes opens up a ton of options (torpedo bombers, submarines, etc.) that were simply not feasible previously.

**Comment by Bubbus** [comment]
These could be cool.  Stretch goal, anyone?

**Comment by Dale1994** [comment]
Yes, this would be pretty smashing! I would also love to see land and sea mines. But that would be another goal.

**Comment by Cre8or** [comment]
Torpedos would need a completely new physics system, because they have to float while in the water. Also, I'm not sure whether making an entity float at a specific depth is even feasible. Either way, this would be a stretch goal if anything, so it's not happening anytime soon.

Also, I'm fairly sure we already have land mines, though they're part of a different addon.

**Comment by ColBeckett** [comment]
just let me know if you decide to add them... i will make necessary models

**Comment by Bubbus** [comment]
Yeah we have landmines in https://github.com/Bubbus/ACF-SWEPs.  They're player-placed, no dispensers (yet?).

Sea mines could be cool.  One thing at a time, it's very easy to have ideas ;)

**Comment by Unicide** [comment]
I've gone ahead and started working on some. They currently float. I'm pretty sure the way I'm doing it doesn't add significant processing time or network traffic but I haven't really dug through ACF enough to know for sure.
